tests/test_dynamo_db_insert_block.py

```diff
@@ -3,28 +3,29 @@
 from nio.common.signal.base import Signal
 from nio.util.support.block_test_case import NIOBlockTestCase
 from nio.modules.threading import spawn
-from ..dynamo_db_block import DynamoDB
+from ..dynamo_db_insert_block import DynamoDBInsert
+from ..dynamo_db_base_block import DynamoDBBase
 from boto.exception import JSONResponseError
 from boto.dynamodb2.fields import HashKey, RangeKey
 
 
-class SaveCounterDynamoDB(DynamoDB):
+class SaveCounterDynamoDB(DynamoDBInsert):
 
     def __init__(self):
         super().__init__()
         self._count = 0
 
-    def _save_signals_to_table(self, table, signals):
+    def execute_signals_query(self, table, signals):
         self._count += 1
-        super()._save_signals_to_table(table, signals)
+        super().execute_signals_query(table, signals)
 
 
-class TestDynamo(NIOBlockTestCase):
+@patch(DynamoDBBase.__module__ + '.connect_to_region')
```

Comment:
I'm ok with patching the other methods here too (create, count, etc) even if a test doesn't necessarily require them to be patched. It's probably safer for adding new tests later